examples/formbot/domain.yml

```diff
@@ -20,7 +20,7 @@ slots:
   cuisine:
     type: unfeaturized
     auto_fill: false
-  num_people:
+  number:
```

Comment:
I think we specifically created slot `num_people` to show how fill slot with different name from the entity